### ISSUE TITLE
Empty arrays in `isPartOf` property

### ISSUE THREAD

**Issue by remibetin** [w3c/wcag-act-rules#315]
Context: https://github.com/w3c/wcag-act-rules/pull/314#discussion_r1887062678

At multiple times in [`equal-access.json`](https://github.com/w3c/wcag-act-rules/blob/main/content-assets/wcag-act-rules/earl/equal-access.json), the `isPartOf` property contains a WCAG success criterion value AND an empty array.

Example:
https://github.com/w3c/wcag-act-rules/blob/cb40747110b802ff7c38140c54235951ad7f192f/content-assets/wcag-act-rules/earl/equal-access.json#L3871-L3874

I understand from [EARL Reporting Format – Test Criterion](https://github.com/w3c/wcag-act-rules/blob/main/content/report/earl.md#test-criterion) that `isPartOf` contains "strings, listing the WCAG success criteria that fail when the rule or procedure fails".

What is the purpose of an empty array in this context?

**Comment by daniel-montalvo** [comment]
Hello @tombrunet 

Can you take a look at this to see if there is any missing info here?

**Comment by tombrunet** [comment]
But in our reporting. These were rules that we kept as ARIA / HTML spec issues that were not directly related to success criteria. They should not have shown up. Will be fixed by https://github.com/IBMa/equal-access/pull/2193

**Comment by remibetin** [comment]
Thanks @tombrunet @daniel-montalvo. The fix is included in https://github.com/w3c/wcag-act-rules/pull/321
